docs/api/field_types_reference/binaryfilefield.md

```diff
@@ -26,33 +26,30 @@ Note that both `BinaryFile` and `Media` Value and Type inherit from the `BinaryB
 |`mimeType`|string|The file's MIME type.|application/pdf|
 |`uri`|string|The binary file's `content/download` URI. If the URI doesn't include a host or protocol, it applies to the request domain.|/content/download/210/2707|
 |`downloadCount`|integer|Number of times the file was downloaded|0|
-|`path`|string|**deprecated**|N/A|
 
-### Hash format
+#### Constructor's hash format
 
 The hash format mostly matches the value object. It has the following keys:
 
-- `id`
-- `path` (for backwards compatibility)
-- `fileName`
-- `fileSize`
-- `mimeType`
-- `uri`
-- `downloadCount`
+- `id` (mandatory, the path to the file to upload into the field)
```

Comment:
As `path`, `id` is BC alias for `inputUri` according to https://github.com/ezsystems/ezplatform-kernel/blob/1.3/eZ/Publish/Core/FieldType/BinaryBase/Type.php#L212-L213